src/domain/ecoverse/ecoverse.service.ts

```diff
@@ -159,6 +163,17 @@ export class EcoverseService {
     }
   }
 
+  async getTemplates(): Promise<ITemplate[]> {
+    try {
+      const ecoverseId = await this.getEcoverseId();
+      const templates = await this.templateService.getTemplates(ecoverseId);
+      return templates;
+    } catch (e) {
+      // this.eventDispatcher.dispatch(events.logger.error, { message: 'Something went wrong in getMembers()!!!', exception: e });
+      throw e;
```

Comment:
No point to rethrow, with the entry above commented / removed and no context added to the Error object, this is pointless. Unhandled exceptions are handled by Nest anyway.

Reply:
Agreed, removed all the catching / rethrowing which was from a while back